src/main/java/bio/terra/cli/command/app/passthrough/Gcloud.java

```diff
@@ -15,4 +48,31 @@ public String getExecutableName() {
   public String getInstallationUrl() {
     return "https://cloud.google.com/sdk/docs/install";
   }
+
+  @Override
+  protected void executeImpl() {
+    workspaceOption.overrideIfSpecified();
+    command.add(0, getExecutableName());
```

Comment:
I assume this can be deleted if you put back `subcommands.get("gcloud").setStopAtPositional(true);` ? Also line 76.

Reply:
Hi Melissa, this line of code is to add "gcloud" at the beginning of `command` that is executed at line 76. For example, when user input `terra gcloud builds submit ....`, the command get parsed and will stores `["builds", "submit", "...."]` into `command`. Then a "gcloud" string is added at the front so that `["gcloud", "builds", "submit", "...."]` can be executed. The same behavior can be found at Line 48 in Git.java and Line 56 in ToolCommand.java.